packages/ag-charts-enterprise/src/features/annotations/annotations.ts

```diff
@@ -479,10 +479,13 @@ export class Annotations extends _ModuleSupport.BaseModuleInstance implements _M
         this.active = hovered;
         toolbarManager.toggleGroup('annotations', 'annotationOptions', this.active != null);
 
-        if (this.active != null) {
+        if (this.active == null) {
+            this.ctx.tooltipManager.unsuppressTooltip('annotations');
+        } else {
             const node = annotations.nodes()[this.active];
             node.toggleActive(true);
             this.ctx.toolbarManager.changeFloatingAnchor('annotationOptions', node.getAnchor());
+            this.ctx.tooltipManager.suppressTooltip('annotations');
```

Comment:
Using `updateTooltip('annotations')` with null content gets overriden immediately by the updated tooltip meta and content from `Chart.handlePointerTooltip()` when a new pointer event occurs.

We could alternatively change the tooltip state to keep the states in the order they are first created, but this would differ from the rest of our states where it is sorted by last updated.

Reply:
e.g.

```diff
+++ b/packages/ag-charts-community/src/util/stateTracker.ts
@@ -1,13 +1,14 @@
 export class StateTracker<T, K = string> extends Map<K, T> {
     constructor(
         protected readonly defaultValue?: T,
-        protected readonly defaultState?: K
+        protected readonly defaultState?: K,
+        protected readonly sort: 'created' | 'updated' = 'updated'
     ) {
         super();
     }

     override set(key: K, value?: T) {
-        this.delete(key); // removed even if re-set to make sure we're last
+        if (this.sort === 'updated') this.delete(key); // removed even if re-set to make sure we're last
```

Reply:
Current approach seems reasonable for now - let's keep an eye out for any gotchas 👍